packages/orchestration/test/supports.ts

```diff
@@ -110,11 +110,11 @@ export const commonSetup = async t => {
   );
   await setupIBCProtocol();
 
-  const { makeOrchestrationKit } = prepareOrchestrationTools(
+  const makeOrchestrationKit = prepareCosmosInterchainService(
     rootZone.subZone('orchestration'),
     vowTools,
   );
-  const { public: orchestration } = makeOrchestrationKit({ portAllocator });
+  const orchestration = makeOrchestrationKit({ portAllocator });
```

Comment:
```suggestion
  const makeCosmosInterchainService = prepareCosmosInterchainService(
    rootZone.subZone('orchestration'),
    vowTools,
  );
  const cosmosInterchainService = makeCosmosInterchainService({ portAllocator });
```

Reply:
I updated bootstrap to match the real one but left `privateArgs` since those are arbitrary